src/content/configuration/output.md

```diff
@@ -750,6 +750,49 @@ module.exports = {
 };
 ```
 
+`libraryTarget: 'system'` - This will expose your library as a [System.register](https://github.com/systemjs/systemjs/blob/master/docs/system-register.md)
```

Comment:
Suggest we wrap real code call references with \`
```suggestion
`libraryTarget: 'system'` - This will expose your library as a [`System.register`](https://github.com/systemjs/systemjs/blob/master/docs/system-register.md)
```